pkg/enqueue/Consumption/FallbackSubscriptionConsumer.php

```diff
@@ -13,21 +13,16 @@
      *
      * @var array
      */
-    private $subscribers;
+    private $subscribers = [];
 
     /**
      * @var int
      */
     private $idleTime = 0;
 
-    public function __construct()
-    {
-        $this->subscribers = [];
-    }
-
     public function consume(int $timeout = 0): void
     {
-        if (empty($this->subscribers)) {
+        if (!$subscriberCount = \count($this->subscribers)) {
```

Comment:
please revert the change, the var could be initialized after this if condition.

Reply:
Simple question: isn't `$this->subscribers` always an array?

Reply:
always

Reply:
So are you sure you want to keep the `empty` check, as `count` should do the same, and the advantage is that it's allowed to initialize the variable at the same time ;)